tests/tests_pytorch/conftest.py

```diff
@@ -190,6 +190,21 @@ def tmpdir_server(tmpdir):
         server.shutdown()
 
 
+@pytest.fixture
```

Comment:
Why add this to conftest if it's only used in one test file?

Reply:
You're correct, for some reason I thought that fixture should be only in `conftest`. Moved it to the appropriate file